src/index.js

```diff
@@ -1,6 +1,7 @@
 const puppeteer = require('puppeteer');
 const adblocker = require('@cliqz/adblocker-puppeteer');
 const cross_fetch = require('cross-fetch');
+const clientVersion = '1.0.3'
```

Comment:
@geronsv [This line](https://github.com/zytedata/zyte-smartproxy-puppeteer/blob/b02dd1633d4b222bfc115bbde0e2708038bf8dc7/package.json#L3) should be updated as well.
Also, maybe use the value from `package.json` instead of declaring a new variable here? ref:
- https://stackoverflow.com/a/22339262/5128406
- https://docs.npmjs.com/cli/v6/using-npm/scripts#packagejson-vars

Reply:
Thank you, Shashank!
This is much better.